### ISSUE TITLE
Call to a member function __setCookie() on null

### ISSUE THREAD

**Issue by n3ISe** [artisaninweb/laravel-soap#60]
`FatalThrowableError in Service.php line 330:
Call to a member function __setCookie() on null
`
i get this when i try to add cookie `->cookie('randomName','blablabla')`

```
public function cookie($name, $value)
  {
    $this->client->__setCookie($name, $value);  // <--- line 330

    return $this;
  }
```

**Comment by n3ISe** [comment]
anyone?

**Comment by artisaninweb** [comment]
Can you post the code you are using here?

**Comment by n3ISe** [comment]
```
SoapWrapper::add(function ($service) {
			   $service
			   ->name('testing')
			   ->wsdl('http://example.asmx?WSDL')                                                   
			   ->trace(true)
			  ->cookie('randomName',$_SESSION['sessionId']);// Optional: (parameters: $name,$value);
			 });
		
		// Using the added service
		SoapWrapper::service('testing', function ($service) {
			
			print_r($service->call('somefunction', []));
			print_r($service->getLastRequestHeaders());
		});
```

**Comment by artisaninweb** [comment]
It's a bug, will fix this in a next version.
For now if i'm correct you can use the model extention.

**Comment by n3ISe** [comment]
noted. 

BTW, can u guide me how to use model extention?

**Comment by artisaninweb** [comment]
Fixed it in version 0.3.*
